makeExportsHot.js

```diff
@@ -29,10 +29,15 @@ function makeExportsHot(m, React) {
       continue;
     }
 
-    if (!isReactClassish(freshExports[key], React)) {
+    try {
```

Comment:
I think it would be nicer to put `try`/`catch` around  something like `var maybeReactClass = freshExports[key]` to make it clear where we might expect the error.